.gitignore/test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [ ] The **.gitignore** is not a **folder** but a **_file_** see the screenshots 👇.

![Capture web_14-9-2021_16311_github com](https://user-images.githubusercontent.com/61562626/133283047-4e816692-8160-4c03-b9bd-2a0566345a49.jpeg)
![mmmm](https://user-images.githubusercontent.com/61562626/133283052-999759ec-8ab4-42ce-8c4a-d51c87cc4f09.jpeg)
![lmlml](https://user-images.githubusercontent.com/61562626/133283170-e3d0106b-223d-4202-8d85-84d6fb1f868a.jpeg)